.dockerignore

```diff
@@ -1,3 +1,2 @@
 .git/
 cargo/
-target/
```

Comment:
Will this change impact people using the docker via mob prompt for dev as well?

Reply:
I don't believe so, since I think mob prompt uses a volume to share that directory.